app/components/how-do-you-feel.jsx

```diff
@@ -0,0 +1,95 @@
+//https://github.com/EnCiv/civil-pursuit/issues/62
+import React, { useState } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+
+function Howdoyoufeel(props) {
+    const {
+        className = '',
+        onDone = () => { },
+        title = '',
+        disabled = false,
+        disableOnClick = false,
+        children,
+        ...otherProps
+    } = props
+    const [isDisabled, setIsDisabled] = useState(disabled)
+    const classes = buttonStyles()
+    const combinedClassName = cx(classes.buttonBase, className)
+
+    return (
+        <div>
```

Comment:
className from props and ...otherProps should be applied only to the outer most tag. meaning this div.  This is so parents of this component can treat this component like a div and handle conditions we haven't considered yet.